packages/mdc-tooltip/README.md

```diff
@@ -0,0 +1,180 @@
+<!--docs:
+title: "Tooltips"
+layout: detail
+section: components
+iconId: tooltip
+path: /catalog/tooltips
+-->
+
+# Tooltips
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/tooltip.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/tooltip.png" width="376" alt="Tooltip screenshot">
+  </a>
+</div>-->
+
+MDC Tooltip provides the user with short descriptions that appear on hover, focus and touch of the controller element.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.google.com/components/tooltips.html">Material Design guidelines: Tooltips</a>
+  </li>
+  <li class="icon-list-item icon-list-item--link">
+    <a href="https://material-components-web.appspot.com/tooltip.html">Demo</a>
+  </li>
+</ul>
+
+## Installation
+
+```
+npm install --save @material/tooltip
+```
+
+## Usage
+
+```html
+<button class="mdc-fab material-icons" aria-label="Favorite">
+  <span class="mdc-fab__icon">
+    arrow_downward
+  </span>
+  <span class="mdc-tooltip">My Tooltip</span>
```

Comment:
Since tooltips aren't supposed to contain rich content/images, I wonder if it would make sense to use the `title` attribute instead?  I know the MDL library uses your same technique, though.

Reply:
Do you mean something like this:
```html
<span class="mdc-fab__icon" data-mdc-tooltip="My Tooltip">
  arrow_downward
</span>
```
and then using CSS to create the tooltip? If so, there has been a short discussion about it here: https://github.com/material-components/material-components-web/issues/1741

If not, could you maybe show a short example of what you have in mind?

Reply:
Hey @j-o-d-o, I had something like this in mind: